libcst/codemod/commands/rename.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# pyre-strict
+import argparse
+from typing import Optional, Sequence, Union
+
+import libcst as cst
+from libcst.codemod import CodemodContext, VisitorBasedCodemodCommand
+from libcst.codemod.visitors import AddImportsVisitor
+from libcst.helpers import get_full_name_for_node
+from libcst.metadata import QualifiedName, QualifiedNameProvider, QualifiedNameSource
+
+
+class RenameCommand(VisitorBasedCodemodCommand):
+    """
+    Rename all instances of a variable or import.
```

Comment:
The rename command works for all Python objects (module, class, function, name). I _think_ this is what you intend with "import" but different phrasing could be more clear.